mods/ts/rules/world.yaml

```diff
@@ -54,6 +54,7 @@
 		PipColor: Red
 		AllowedTerrainTypes: Clear
 		AllowUnderActors: true
+		AllowUnderBuildings: true
```

Comment:
In the original veins grew under buildings and destroyed them quite rapidly, so I guess we will probably want this as soon as veins are functional